reference/5.1/Microsoft.PowerShell.Core/About/about_Module_Manifest.md

```diff
@@ -0,0 +1,1714 @@
+
+```yaml
+Type: System.Version
+Aliases:
+Required: True
+Default value: 1.0
+Accept wildcard characters: False
+```
+
+This setting specifies the version of the module. When multiple versions of a
+module exist on a system, the latest version is loaded by default when you use
+`Import-Module`.
+
+PowerShellGet uses the version for module installation. You can use the
+`#Requires` statement in a script or the **RequiredModules** setting of another
+module manifest to specify minimum, maximum, and ranges of compatible versions.
+
+#### When ModuleVersion is a valid string
+
+In this example, the specified string is converted automatically to a
+**System.Version** object.
+
+```powershell
+@{
+    ModuleVersion = '1.2.3'
+}
+```
+
+When you import the module and inspect the **Version** property, note that it is
+a **System.Version** object and not a string:
+
+```powershell
+$ExampleModule = Import-Module example.psd1
+$ExampleModule.Version
+$ExampleModule.Version.GetType().Name
+```
+
+```Output
+Major  Minor  Build  Revision
+-----  -----  -----  --------
+1      2      3      -1
+
+Version
+```
+
+Because PowerShell automatically converts the value for you, it is not necessary
+to cast the string to **System.Version** yourself.
+
+#### When ModuleVersion cannot be converted to System.Version
+
+Because the string `a.b.c` specified in this example cannot be converted to a
+**System.Version** object, this setting causes both `Test-ModuleManifest` and
+`Import-Module` to return an error.
+
+```powershell
+@{
+    ModuleVersion = 'a.b.c'
+}
+```
+
+### CompatiblePSEditions
+
+```yaml
+Type: System.String[]
+Aliases:
+Accepted values: Desktop, Core
+Required: False
+Default value: None
+Accept wildcard characters: False
+```
+
+Specifies the module's compatible PSEditions. If not set, the module is expected
+to be compatible for all PSEditions. For information about PSEdition, see
+[Modules with compatible PowerShell Editions](/powershell/scripting/gallery/concepts/module-psedition-support).
+
+#### When CompatiblePSEditions is unset
+
+This module can be imported in both **Desktop Edition** and **Core Edition**
+sessions, as well as older versions of Windows PowerShell.
+
+```powershell
+@{
+    # CompatiblePSEditions = @()
+}
+```
+
+#### When CompatiblePSEditions is set to Desktop
+
+This module can only be imported in Windows PowerShell sessions where the
+`$PSEdition` automatic variable's value is `Desktop`.
+
+```powershell
+@{
+    CompatiblePSEditions = @('Desktop')
+}
+```
+
+#### When CompatiblePSEditions is set to Core
+
+This module can only be imported in sessions where the `$PSEdition` automatic
+variable's value is `Core`.
+
+```powershell
+@{
+    CompatiblePSEditions = @('Core')
+}
+```
+
+#### When CompatiblePSEditions is set to Desktop and Core
+
+This module can only be imported in sessions where the `$PSEdition` automatic
+variable exists.
+
+```powershell
+@{
+    CompatiblePSEditions = @('Desktop', 'Core')
+}
+```
```

Comment:
Ditch the H4s. Make the description more imperative. Don't need to show all 4 possible combinations. Just explain that "Undefined is the same as `@('Desktop', 'Core')`.

An empty array is allowed. Do we know if that is the same as both? Or what does it mean to be 'none'?